src/mousetracker.js

```diff
@@ -318,6 +318,20 @@
         },
 
         /**
+         * Returns {Array.<OpenSeadragon.MouseTracker>} excluding the given pointer device type
+         * @function
+         * @param {String} type - The pointer device type: "mouse", "touch", "pen", etc.
+         * @returns {Array.<OpenSeadragon.MouseTracker>}
+         */
+        getActivePointersListExceptType: function ( type ) {
+            var delegate = THIS[ this.hash ];
+
+            return delegate.activePointersLists.filter(function(pointersList) {
```

Comment:
We're still holding onto IE8 support (though maybe not much longer... see https://github.com/openseadragon/openseadragon/issues/373) so we should probably do this filter the old-fashioned way (with a for loop).